github/messages_test.go

```diff
@@ -82,6 +83,54 @@ func TestValidatePayload(t *testing.T) {
 	}
 }
 
+func TestValidateFormPayloadGET(t *testing.T) {
```

Comment:
Rename the test to `TestValidatePayload_FormGet` for more consistent test naming.